Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1010,10 +1010,17 @@
 #define ENABLE_WIRELESS_PLAYBACK_TARGET_AVAILABILITY_API 1
 #endif
 
-#if !defined(ENABLE_WK_WEB_EXTENSIONS) && (PLATFORM(MAC) || PLATFORM(MACCATALYST) || PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR) || PLATFORM(VISION))
+#if !defined(ENABLE_WK_WEB_EXTENSIONS) && (PLATFORM(MAC) || PLATFORM(MACCATALYST) || PLATFORM(IOS) || PLATFORM(VISION))
```

Comment:
Did you mean to turn off the feature flag for the iOS sim?

Reply:
If so, this should be mentioned in the commit message, but I'm worried about what it would do for the tests.

Reply:
Sim is already covered by `PLATFORM(IOS)`, it was a mistake to have it originally.